recipes/pyomo.solvers/meta.yaml

```diff
@@ -0,0 +1,36 @@
+# Do metapackages have a version?
+# Do metapackages have a license?
```

Comment:
Can you remove these comments and the blank lines?

PS: the answer is yes and yes :smile: